proto/osmosis/epochs/genesis.proto

```diff
@@ -7,27 +7,62 @@ import "google/protobuf/timestamp.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v7/x/epochs/types";
 
+// EpochInfo is a struct that describes the data going into
+// a timer defined by the x/epochs module.
 message EpochInfo {
+  // Identifier is a unique reference to this particular timer.
   string identifier = 1;
+  // Start time is the time at which the timer first ever ticks.
+  // If start time is in the future, the epoch will not begin until the start
+  // time.
   google.protobuf.Timestamp start_time = 2 [
     (gogoproto.stdtime) = true,
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"start_time\""
   ];
+  // Duration is the time in between epoch ticks.
+  // In order for intended behavior to be met, duration should
+  // be greater than the chains expected block time.
+  // Duration must be non-zero.
   google.protobuf.Duration duration = 3 [
     (gogoproto.nullable) = false,
     (gogoproto.stdduration) = true,
     (gogoproto.jsontag) = "duration,omitempty",
     (gogoproto.moretags) = "yaml:\"duration\""
   ];
+  // current_epoch is the current epoch number, or in other words,
```

Comment:
Good point, hadn't considered this. 

This one is odd, because every language will generate this one differently. I guess we should make the langauge-specific proto generators really handle updating the comments

I guess since were the main consumer for now, but I think longer term we should revert back

Reply:
I guess if we have to revert this, let's adapt the long-term desired approach right away and create an issue to figure out Go-specific generations. Wdyt?

Reply:
SGTM, just on our list of tooling / automation things to do, this seems incredibly low